tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-cloudfoundry/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -0,0 +1 @@
+configuration-properties.classes=
```

Comment:
Well, if really there were no properties, then we should not have a file at all.

BUT, IMO the `CloudFoundryDeploymentProperties` and `CloudFoundryConnectionProperties` used by this app are WL properties.